.github/workflows/pr-title-linter-and-linker.yml

```
@@ -0,0 +1,82 @@
+name: "Lint and Link PR title"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+jobs:
+  lint:
+    name: Validate PR title And link Jira Issue
+    runs-on: ubuntu-22.04
+    env:
+      JIRA_COMMENT_REGEX: "^.*Jira.*"
+    steps:
+      - uses: Slashgear/action-check-pr-title@860e8dc639f8e60335a6f5e8936ba67ed2536890 #v4.3.0
+        id: lint
+        with:
+          regexp: "\\[(#?[A-Z]*-[0-9]*( |, )?){1,}\\]" # Regex the title should match.
```

```suggestion
          regexp: "\\[(#?[A-Z]*-[0-9]*,?){1,}\\]" # Regex the title should match.
```
This should match [ABC-123,ABC-456] instead of [ABC-123, ABC-456] which is what we are used to. Does it make sense to you?